src/System Application/App/SFTP Client/src/dotnet.al

```diff
@@ -0,0 +1,26 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.SFTPClient;
+
+dotnet
+{
+    assembly("Renci.SshNet")
```

Comment:
These actually also need to get added to the dotnet.al file in the System Application (in the DotNet module), as the system application is built as one.

Reply:
Oh yes! I updated the PR with the Dotnet Aliases.